node_cli/main.js

```diff
@@ -0,0 +1,80 @@
+const inquirer = require("inquirer");
+const tasks = require("./tasks");
+const { logout } = require("./realm/realmAuth");
+const index = require("./index");
+
+async function mainMenu(authedUser) {
```

Comment:
This is an async function, so you can `await` instead of using a promise chain

```js
async function mainMenu(authedUser) {
  try {
    const answers = await inquirer.prompt(...);
    switch(answers.mainMenu) {
      ...
    }
  } catch (err) {
    index.output(err, "error");
    return;
  }
}
```